superset-frontend/src/components/Form/Form.stories.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useState } from 'react';
+import LabeledErrorBoundInput, { Form } from '.';
+
+export default {
+  title: 'Form',
+  component: Form,
+};
+
+export const InteractiveForm = () => {
+  const [checkErrorMessage, setCheckErrorMessage] = useState('');
+
+  const validateFunctionality: (value: any) => void = value => {
+    if (value.target.value.includes('success')) {
+      setCheckErrorMessage('');
+    } else {
+      setCheckErrorMessage('Type success in the text bar');
+    }
+  };
+
+  return (
+    <LabeledErrorBoundInput
```

Comment:
I would suggest renaming the file to this since we changed the naming of the component.

Reply:
Fixed at [`this commit`](https://github.com/apache/superset/pull/14530/commits/7f2e7534091d695a41f75cd1e7dc9663acd5c8e2)

Reply:
And [`this one`](https://github.com/apache/superset/pull/14530/commits/f46dc8841865dbd9deb91ef93112f6833d78ce3f)